Sources/ArgumentParser/Completions/CompletionsGenerator.swift

```diff
@@ -164,6 +164,24 @@ extension ParsableCommand {
   }
 }
 
+extension [ParsableCommand.Type] {
+  var positionalArguments: [ArgumentDefinition] {
+    guard let command = last else {
+      return []
+    }
+    return ArgumentSet(command, visibility: .default, parent: nil)
+      .filter(\.isPositional)
+  }
+
+  /// Include default 'help' subcommand in nonempty subcommand list if & only if
+  /// no help subcommand already exists.
+  mutating func addHelpSubcommandIfMissing() {
+    if !isEmpty && allSatisfy({ $0._commandName != "help" }) {
```

Comment:
Let's rewrite to flip the polarity:
```suggestion
    if !isEmpty && !contains(where: { $0._commandName == "help" }) {
```